docs/source/reference/config-user-env.md

```diff
@@ -30,7 +30,7 @@ system-wide or in a shared environment.
 This installation location should always be in the same environment that
 `jupyterhub-singleuser` itself is installed in, and must be _readable and
 executable_ by your users. If you want users to be able to install additional
-packages, it must also be _writable_ by your users.
+packages, they must also be _writable_ by your users.
```

Comment:
`it` is actually the correct pronoun here, because the statement refers to the `instalation location` that must be `writable` and not to the users. Let's instead be explicit to avoid any future confusion:

```suggestion
packages, the installation location must also be _writable_ by your users.
```

Reply:
Done, thank you!